pkg/tablebackup/tablebackup.go

```diff
@@ -125,45 +129,90 @@ func New(ctx context.Context, cfg *config.Config, tbl message.Identifier, dbCfg
 }
 
 func (t *TableBackup) SaveRawMessage(msg []byte, lsn uint64) (uint64, error) {
```

Comment:
Shall we merge `SaveRawMessage` and `appendDeltaToSegment` as `SaveRawMessage` is not saving anything anymore

Reply:
We could, but I'd like to reduce the number of lines in SaveRawMessage to make it more readable.

Reply:
my concern is `SaveRawMessage` is not saving anything anymore, maybe we should rename it then?